README.md

```diff
@@ -38,8 +38,16 @@ You have two options:
 *   Install globally `eslint` and plugins
 
     *   `$ npm i -g eslint [eslint-plugins]`
-    *   Activate `Use Global Eslint` package option
+    *   Activate `Use Global Eslint` package option in Sublime's config
+    ```
```

Comment:
Exact code like this is _specifically_ not in there, because you should just be using the Settings UI to change it in the first place.

For anyone who feels the need to edit that file manually, they should intimately know the format and how to find the setting values themselves.